### ISSUE TITLE
Looking for Fluent UI Microsoft Icons in SVG (M365 and D365)

### ISSUE THREAD

**Issue by faisal-ijaz** [microsoft/fluentui#17166]
Hi, I am looking for Microsoft Fluent Icons for m365 and d365. I have been able to find most of the m365 icons though I am using some low res ones for the ones I don't have. The D365 can't seem to be anywhere on the web. It would be awesome if someone can help point me in the right direction to obtain svgs for these icons. 

Thanks

**Comment by ecraig12345** [comment]
@tomi-msft

**Comment by tomi-msft** [comment]
Hi @faisal-ijaz where have you looked for the icons? Which icons specifically are you looking for? It sounds like you're looking for branded svg icons, so I would check in `@fluentui/react-icons` in fluentui v7, or `@fluentui/react-icons-mdl2-branded` in fluentui v8. 

If they're not there, then they might be in the [office brand icons](https://developer.microsoft.com/en-us/fluentui#/styles/web/office-brand-icons)

**Comment by faisal-ijaz** [comment]
Hi, sorry for the late response. I am indeed looking for the branded icons. So the ones I am looking for are d365 icons. I have checked and the office brand icons dos not include d365 icons. I was able to find some d365 icons by going to help pages:

https://docs.microsoft.com/en-us/dynamics365/customer-service/
https://docs.microsoft.com/en-us/dynamics365/sales/

Rest I could not find like Marketing, Finance, etc..
